sdk/eventgrid/arm-eventgrid/review/arm-eventgrid.api.md

```diff
@@ -3736,15 +3736,24 @@ export interface StaticDeliveryAttributeMapping extends DeliveryAttributeMapping
     value?: string;
 }
 
-// @public (undocumented)
+// @public
 export interface StaticRoutingEnrichment {
     key?: string;
-    valueType?: StaticRoutingEnrichmentType;
+    valueType: "String";
```

Comment:
I think this is a breaking here, because it's narrowing the valueType scope, can you help check with service team if it works if we put a different value than "String" now? if yes, we should bump major version.